src/Directory.Build.props

```diff
@@ -8,19 +8,20 @@
     <EnablePublicApi>true</EnablePublicApi>
 
     <!-- NuGet packages -->
+    <Version>0.6.0$(VersionSuffix)</Version>
     <IsPackable>true</IsPackable>
     <PackageIconUrl>https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/raw/main/opentelemetry-logo-64x64.png</PackageIconUrl>
```

Comment:
Before publishing, check if we are using the same file as SDK/contrib

Reply:
Definitely, before going public we should review all of the NuGet properties. For instance: PackageIconUrl is actually deprecated (see https://github.com/open-telemetry/opentelemetry-dotnet/pull/325). Related: @pellared suggested using a validation tool to ensure some properties on the NuGet packages.